Sources/SwiftSyntaxBuilder/gyb_generated/Tokens.swift

```diff
@@ -16,181 +16,450 @@ import SwiftSyntax
-  static let `poundAssert` = SyntaxFactory.makePoundAssertKeyword().withTrailingTrivia(.spaces(1))
+  static let `poundAssert` = SyntaxFactory.makePoundAssertKeyword()
+    .withLeadingTrivia(.spaces(1))
+
   /// The `#sourceLocation` keyword
-  static let `poundSourceLocation` = SyntaxFactory.makePoundSourceLocationKeyword().withTrailingTrivia(.spaces(1))
+  static let `poundSourceLocation` = SyntaxFactory.makePoundSourceLocationKeyword()
+    .withLeadingTrivia(.spaces(1))
+
   /// The `#warning` keyword
-  static let `poundWarning` = SyntaxFactory.makePoundWarningKeyword().withTrailingTrivia(.spaces(1))
+  static let `poundWarning` = SyntaxFactory.makePoundWarningKeyword()
+    .withLeadingTrivia(.spaces(1))
+
   /// The `#error` keyword
-  static let `poundError` = SyntaxFactory.makePoundErrorKeyword().withTrailingTrivia(.spaces(1))
+  static let `poundError` = SyntaxFactory.makePoundErrorKeyword()
+    .withLeadingTrivia(.spaces(1))
+
   /// The `#if` keyword
-  static let `poundIf` = SyntaxFactory.makePoundIfKeyword().withTrailingTrivia(.spaces(1))
+  static let `poundIf` = SyntaxFactory.makePoundIfKeyword()
+    .withLeadingTrivia(.spaces(1))
+
   /// The `#else` keyword
-  static let `poundElse` = SyntaxFactory.makePoundElseKeyword().withTrailingTrivia(.spaces(1))
+  static let `poundElse` = SyntaxFactory.makePoundElseKeyword()
+    .withLeadingTrivia(.spaces(1))
+
   /// The `#elseif` keyword
-  static let `poundElseif` = SyntaxFactory.makePoundElseifKeyword().withTrailingTrivia(.spaces(1))
+  static let `poundElseif` = SyntaxFactory.makePoundElseifKeyword()
+    .withLeadingTrivia(.spaces(1))
+
   /// The `#endif` keyword
-  static let `poundEndif` = SyntaxFactory.makePoundEndifKeyword().withTrailingTrivia(.spaces(1))
+  static let `poundEndif` = SyntaxFactory.makePoundEndifKeyword()
+    .withLeadingTrivia(.spaces(1))
+
   /// The `#available` keyword
-  static let `poundAvailable` = SyntaxFactory.makePoundAvailableKeyword().withTrailingTrivia(.spaces(1))
+  static let `poundAvailable` = SyntaxFactory.makePoundAvailableKeyword()
+    .withLeadingTrivia(.spaces(1))
+
   /// The `#fileLiteral` keyword
-  static let `poundFileLiteral` = SyntaxFactory.makePoundFileLiteralKeyword().withTrailingTrivia(.spaces(1))
+  static let `poundFileLiteral` = SyntaxFactory.makePoundFileLiteralKeyword()
+    .withLeadingTrivia(.spaces(1))
+
   /// The `#imageLiteral` keyword
-  static let `poundImageLiteral` = SyntaxFactory.makePoundImageLiteralKeyword().withTrailingTrivia(.spaces(1))
+  static let `poundImageLiteral` = SyntaxFactory.makePoundImageLiteralKeyword()
+    .withLeadingTrivia(.spaces(1))
+
   /// The `#colorLiteral` keyword
-  static let `poundColorLiteral` = SyntaxFactory.makePoundColorLiteralKeyword().withTrailingTrivia(.spaces(1))
+  static let `poundColorLiteral` = SyntaxFactory.makePoundColorLiteralKeyword()
+    .withLeadingTrivia(.spaces(1))
+
+  static let `integerLiteral` = SyntaxFactory.makeIntegerLiteral()
```

Comment:
I don’t think this compiles. `makeIntegerLiteral` has a required `text` parameter. 

Stupid question but are you checking if your code compiles before pushing the changes?

Reply:
Note to my self: Remember to delete `DerivedData` when chugging toolchain, sorry about that!  😞
`SwiftSyntax` started not to compile.. Strange.. But it should be fixed now and build and tests pass locally. 

Pushed to https://github.com/apple/swift/pull/36383 as tests did fail with `{` and `}` because we added trailing space. We can not generalise that.